windows_10_hardening.md

```diff
@@ -489,9 +489,10 @@ In enterprise-managed mode, trusted zones can be defined via the network isolati
 
 #### Windows PowerShell
 
-* ID 2100, ID 2101: Set _Turn on PowerShell Script Block Logging_ to **Enabled**
-* ID 2102: Set _Turn on PowerShell Transcription_ to **Enabled**
-* ID 2103, ID 2104: Remove PowerShell Version 2
+* ID 2100, ID 2101: Set _Turn on PowerShell Module Logging_ to **Enabled**
```

Comment:
Can you keep the ID's of the previous PowerShell checks and increase the ID's for module logging?

Reply:
Reverted the ID numbers.